/**: # wildcard for the node name
  ros__parameters:
    ekf_params:
      initial_state:
        x: 1.0
        y: 1.0
        z: 0.0
        roll: 0.0
        pitch: 0.0
        yaw: 0.0
        dx: 0.0
        dy: 0.0
        dz: 0.0
        droll: 0.0
        dpitch: 0.0
        dyaw: 0.0

      initial_state_covariance:
        # standard deviations
        x: 0.2
        y: 0.2
        z: 0.2
        roll: 0.1
        pitch: 0.1
        yaw: 0.1
        dx: 0.1
        dy: 0.1
        dz: 0.1
        droll: 0.01
        dpitch: 0.01
        dyaw: 0.01

      process_noise:
        # standard deviations
        x: 0.05
        y: 0.05
        z: 0.03
        roll: 0.01
        pitch: 0.01
        yaw: 0.01
        dx: 0.3
        dy: 0.1
        dz: 0.1
        droll: 0.01
        dpitch: 0.01
        dyaw: 0.01

      # vision
      measurement_noise:
        # standard deviations
        distance: 0.1
        yaw: 0.1

      # orientation data
      measurement_noise_orientation:
        # standard deviations
        roll: 0.1
        pitch: 0.1

      dim_meas: 2
      dim_state: 12
